frontend/apps/filemanager/filemanager.lua

```diff
@@ -243,8 +244,19 @@ function FileManager:init()
                     end,
                 }
             },
+            -- a little hack to get visual functionality grouping
+            {},
             {
                 {
+                    text = _("Convert"),
+                    enabled = lfs.attributes(file, "mode") == "file"
+                        and FileManagerConverter.formats_from[require("util").getFileNameSuffix(file)] and true or false,
```

Comment:
nitpick, this line is getting too long, perhaps worth abstracting into a method of FileManagerConverter.